test-install/Vagrantfile

```diff
@@ -0,0 +1,24 @@
+# -*- mode: ruby -*-
```

Comment:
Maybe rename this folder to test_install to match other folders naming. If we add here go files someday they would want to be in test_install folder to feel a bit better 😄